db/migrate/20110327123639_add_priority_to_parameter.rb

```diff
@@ -1,8 +1,7 @@
 class AddPriorityToParameter < ActiveRecord::Migration
   def up
     add_column :parameters, :priority, :integer
-    Parameter.reset_column_information
```

Comment:
Might this still be required? The column information may be cached, and the rake task still relies on it. (As may later migrations.)